src/webgl/camera.js

```diff
@@ -173,31 +181,36 @@ p5.RendererGL.prototype.camera = function(
  * @example
  * <div>
  * <code>
- * //drag mouse to toggle the world!
- * //you will see there's a vanish point
+ * //drag the mouse to look around!
+ * //you will see there's a vanishing point
  * function setup() {
  *   createCanvas(100, 100, WEBGL);
  *   var fov = 60 / 180 * PI;
- *   var cameraZ = height / 2.0 / tan(fov / 2.0);
- *   perspective(60 / 180 * PI, width / height, cameraZ * 0.1, cameraZ * 10);
+ *   perspective(fov, width / height, 0.1, 500);
  * }
  * function draw() {
  *   background(200);
  *   orbitControl();
- *   for (var i = -1; i < 2; i++) {
- *     for (var j = -2; j < 3; j++) {
- *       push();
- *       translate(i * 160, 0, j * 160);
- *       box(40, 40, 40);
- *       pop();
- *     }
- *   }
+ *   normalMaterial();
+ *
+ *   rotateX(-0.3);
+ *   rotateY(-0.2);
+ *   translate(0, 0, -50);
+ *
+ *   push();
+ *   translate(-15, 0, sin(frameCount / 30) * 95);
+ *   box(30);
+ *   pop();
+ *   push();
+ *   translate(15, 0, sin(frameCount / 30 + PI) * 95);
+ *   box(30);
+ *   pop();
  * }
  * </code>
  * </div>
  *
  * @alt
- * colored 3d boxes toggleable with mouse position
+ * colored 3D boxes move back and forth, rotating as mouse is dragged.
```

Comment:
this is good, I think we can add "two" to the beginning here for just a little more detail.